javascripts/tileCollection.js

```diff
@@ -0,0 +1,57 @@
+function TileCollection(direction) {
+  // direction will be keypress values -- 37: left, 39: right, 38: up, or 40: down
```

Comment:
To make life easier, you could define a constant object so you don't have to deal with numbers:
```
DIRECTIONS = {
  LEFT: 37,
  RIGHT: 39,
  UP: 38,
  DOWN: 40
}
```

Now whenever you need to talk about direction, you can do something like:
```
if (direction == DIRECTION.LEFT || direction == DIRECTION.RIGHT)
```